client/catapult/extensions/zeromq/tests/ZeroMqEntityPublisherTests.cpp

```diff
@@ -126,13 +126,13 @@ namespace catapult { namespace zeromq {
 	}
 
 	TEST(TEST_CLASS, CanUseCustomListenInterface_IPv4) {
-		AssertCanUseCustomListenInterface("0.0.0.0");
-		AssertCanUseCustomListenInterface("127.0.0.1");
+		AssertCanUseCustomListenInterface("0.0.0.0", "127.0.0.1");
+		AssertCanUseCustomListenInterface("127.0.0.1", "127.0.0.1");
 	}
 
 	TEST(TEST_CLASS, CanUseCustomListenInterface_IPv6) {
-		AssertCanUseCustomListenInterface("::");
-		AssertCanUseCustomListenInterface("::1");
+		AssertCanUseCustomListenInterface("::", "::1");
```

Comment:
Not sure if we should keep this test scenario for linux?
```
AssertCanUseCustomListenInterface("::", "::");
```

Reply:
probably not, tbh I'm not sure what was reasoning behind it